### PR TITLE
fix(ekf_localizer): rename biased pose topics

### PR DESCRIPTION
**Pull request by IshitaTakeshi**
## Description

The current topic names such as `pose_without_yawbias` imply publishing poses without yaw bias, but actually, it has the bias. So I renamed these topics to explicitly indicate that they have the bias.

## Related links

On the current code, there's no bias correction when creating the corresponding message. 

https://github.com/autowarefoundation/autoware.universe/blob/57c4b1fbc95748d59f0f77b1ac84a2c976f17fb8/localization/ekf_localizer/src/ekf_localizer.cpp#L233

## Tests performed

There are no tests performed for this PR but I checked the behavior on the logging simulator. 

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: localization/ekf_localizer/src/ekf_localizer.cpp [hunk 1]
```diff
@@ -680,7 +679,7 @@ void EKFLocalizer::publishEstimateResult()
   pub_pose_cov_->publish(pose_cov);
 
   geometry_msgs::msg::PoseWithCovarianceStamped pose_cov_no_yawbias = pose_cov;
```
Comment (kminoda) [inline localization/ekf_localizer/src/ekf_localizer.cpp:681]:
```suggestion
  geometry_msgs::msg::PoseWithCovarianceStamped biased_pose_cov = pose_cov;
```

File: localization/ekf_localizer/include/ekf_localizer/ekf_localizer.hpp [hunk 2]
```diff
@@ -203,7 +203,7 @@ class EKFLocalizer : public rclcpp::Node
   std::queue<PoseInfo> current_pose_info_queue_;      //!< @brief current measured pose
   geometry_msgs::msg::PoseStamped current_ekf_pose_;  //!< @brief current estimated pose
   geometry_msgs::msg::PoseStamped
-    current_ekf_pose_no_yawbias_;  //!< @brief current estimated pose w/o yaw bias
+    current_biased_ekf_pose_;  //!< @brief current estimated pose w/o yaw bias
```
Comment (kminoda) [inline localization/ekf_localizer/include/ekf_localizer/ekf_localizer.hpp:206]: The comment should also be fixed. How about something like this?
```suggestion
    current_biased_ekf_pose_;  //!< @brief current estimated pose without yaw bias correction
```

Comment (IshitaTakeshi) [reply localization/ekf_localizer/include/ekf_localizer/ekf_localizer.hpp:206]: Looks good!

File: localization/ekf_localizer/src/ekf_localizer.cpp (hunk 1 above)
Comment (IshitaTakeshi) [reply localization/ekf_localizer/src/ekf_localizer.cpp]: Resolved in https://github.com/autowarefoundation/autoware.universe/pull/1787/commits/71f7c37e517e7ae4938af323588142b30ef15c95

File: localization/ekf_localizer/src/ekf_localizer.cpp [hunk 3]
```diff
@@ -89,10 +89,9 @@ EKFLocalizer::EKFLocalizer(const std::string & node_name, const rclcpp::NodeOpti
   pub_twist_cov_ = create_publisher<geometry_msgs::msg::TwistWithCovarianceStamped>(
     "ekf_twist_with_covariance", 1);
   pub_yaw_bias_ = create_publisher<tier4_debug_msgs::msg::Float64Stamped>("estimated_yaw_bias", 1);
-  pub_pose_no_yawbias_ =
-    create_publisher<geometry_msgs::msg::PoseStamped>("ekf_pose_without_yawbias", 1);
+  pub_biased_pose_ = create_publisher<geometry_msgs::msg::PoseStamped>("ekf_biased_pose", 1);
   pub_pose_cov_no_yawbias_ = create_publisher<geometry_msgs::msg::PoseWithCovarianceStamped>(
```
[the remark this answers is not in the document]
Comment (IshitaTakeshi) [reply localization/ekf_localizer/src/ekf_localizer.cpp]: Resolved in https://github.com/autowarefoundation/autoware.universe/pull/1787/commits/71f7c37e517e7ae4938af323588142b30ef15c95

**Comment by YamatoAndo** [comment]
@IshitaTakeshi Thanks for the refactoring.
Could you please submit a PR for [autoware_launch](https://github.com/tier4/autoware_launch) as well?

https://github.com/tier4/autoware_launch/blob/awf-latest/localization_launch/launch/pose_estimator/pose_estimator.launch.xml#L7

https://github.com/tier4/autoware_launch/blob/awf-latest/localization_launch/launch/pose_twist_fusion_filter/pose_twist_fusion_filter.launch.xml#L14-L15

**Review by YamatoAndo — APPROVED** [review]

**Pull request merged**